qemu/tests/fio_perf.py

```diff
@@ -248,20 +248,20 @@ def fio_install(tarball):
                     io_exits_a = int(process.system_output("cat /sys/kernel/debug/kvm/exits"))
                     vm.copy_files_from(guest_result_file, data_dir.get_tmp_dir())
                     fio_result_file = os.path.join(data_dir.get_tmp_dir(), "fio_result")
-                    o = process.system_output("egrep '(read|write)' %s" % fio_result_file)
+                    o = process.system_output("egrep '(read|write)' %s" % fio_result_file).decode()
                     results = re.findall(pattern, o)
-                    o = process.system_output("egrep 'lat' %s" % fio_result_file)
+                    o = process.system_output("egrep 'lat' %s" % fio_result_file).decode()
                     laten = re.findall(r"\s{5}lat\s\((\wsec)\).*?avg=[\s]?(\d+(?:[\.][\d]+)?).*?", o)
-                    bw = float(utils_misc.normalize_data_size(results[0][0]))
-                    iops = int(results[0][1])
+                    bw = float(utils_misc.normalize_data_size(results[0][1]))
```

Comment:
Please try to use `normalize_data_size` from utils_numeric.py, as `utils_misc.normalize_data_size` has bugs.

Reply:
@hereischen Updated